src/cnlpt/train_system.py

```diff
@@ -720,7 +742,8 @@ def compute_metrics_fn(p: EvalPrediction):
         return compute_metrics_fn
 
     # Initialize our Trainer
-    training_args.load_best_model_at_end = True
```

Comment:
@tmills I removed this line because it makes it impossible to do prediction without training; it looks for a model checkpoint in the output dir, but when do_train = False, that doesn't exist.